app/adapters/plan.js

```diff
@@ -1,3 +1,23 @@
 import V3Adapter from 'travis/adapters/v3';
+import { inject as service } from '@ember/service';
+import { reads } from '@ember/object/computed';
 
-export default V3Adapter.extend();
+export default V3Adapter.extend({
+  accounts: service(),
+  account: reads('accounts.user'),
+
+  buildURL(modelName, id, snapshot, requestType) {
+    let url = this._super(...arguments);
+    let route = '/plans_for/user';
+    if (this.account.type === 'organization') {
```

Comment:
`this.account` will never be `organization`, because it's a currently logged in user: https://github.com/travis-ci/travis-web/blob/master/app/services/accounts.js#L15 and there's no way to authorize under organization account (as on GitHub too). It should not be critical for release as long as it'll just always work in `user` flow.